specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/examples/createMcppPurchase.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+      "resourceGroupName": "McppPurchases",
+      "api-version": "2023-02-09-preview",
+      "subscriptionId": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05",
+      "resourceName": "microsoftactionpack",
+      "body": { 
+        "productCode": "test product"            
+    }
+  },          
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.CloudPartnerProgram/membership/microsoftactionpack",
+        "name": "microsoftactionpack",
+        "type": "Microsoft.CloudPartnerProgram/membership",
+        "properties": { 
+          "orderNo": "2302319360",
+          "orderDateTime": "2023-01-23T07:03:56.6279105Z",
+          "status": "AwaitingPayment",
+          "operationDetails":  { 
```

Comment:
You shouldn't have an "operationDetails" property on your resource... Please make sure we follow the standard async pattern for REST operations: https://armwiki.azurewebsites.net/rpaas/async.html

Reply:
Also look at how ARM uses a "provisioningState" property to capture the state of an ARM resource... Maybe that was your intent.

Reply:
yes, the purpose was to capture the current state of the purchase (ex reason for failure in case it fails etc) as its an async operation.  Will change it to provisioing state.  But is there a restriction on what kind of information we are allowed to share here ?

Reply:
You could return the effect of FirstChargePolicy as a state on your resource, yes.

More info on provisioningState itself: https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#provisioningstate-property